Lang/Spanish.pj.Lang

```diff
@@ -177,7 +178,7 @@
 #320#  "Seleccionar directorio de ROMs actual"
 
 //Messages
-#340#  "¿ROM Dañado? Usa GoodN64 y verifica que la RDB esté actualizada."
+#340#  "¿ROM Dañada? Usa GoodN64 y comprueba por una RDB actualizada."

```

Comment:
The main issue with this change is "por".
You can "verify for" in English, or maybe "test for".
But you cannot "comprobar por" in Spanish.

Suggestion: use "comprueba que la RDB esté actualizada".